README.md

```diff
@@ -1 +1,98 @@
-# books-website
\ No newline at end of file
+# books-website
+
+# Project Name : Awsome Books ♻️
+
+This is  Pair to Pair project to help you find and remember the best books to read.
+
+# Screenshots
+- Desktop Layout
+
+![Desktop_Layout](assets/img/desktop.png)
+
+- Mobile Layout
+
+![Mobile_Layout](assets/img/mobile.png)
+
+
+
+Additional description about the project and its features.
```

Comment:
- [ ] [OPTIONAL] Kindly put a screenshot of your project or remove this section as it now makes your README not professional.

Reply:
Thank you @gemmen29 we removed the section from README.